### DISCUSSION TITLE
Send audio voice

### DISCUSSION THREAD

**Discussion by maxHSG** [netflie/whatsapp-cloud-api#51] [Q&A] [answered]
Is it possible to send a voice audio instead of the attachment audio?

![Captura de Tela 2022-10-11 às 14 55 00](https://user-images.githubusercontent.com/30557290/195177114-23c8340f-2e00-4d41-af62-65a31ea86fc9.png)

**Comment by aalbarca** [comment, marked as answer]
Hi @maxHSG 

2.x version has been released! https://github.com/netflie/whatsapp-cloud-api/discussions/75 and you can now upload audio files to WhatsApp servers.
